config/application.rb

```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
 
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{yml}')]
+
+    routes.default_url_options[:host] = Figaro.env.mailer_domain_name
```

Comment:
ah! good to know @monfresh  -- does that mean we should remove the protocol from `mailer_domain_name`